packages/components/Badge/src/PresenceBadge/PresenceBadge.tsx

```diff
@@ -57,7 +57,7 @@ export const PresenceBadge = compose<PresenceBadgeType>({
       const path = getIconPath(status, isOutOfOffice);
 
       return (
-        <Slots.root {...mergedProps}>
+        <Slots.root {...mergedProps} accessibilityLabel={status} accessible={true}>
```

Comment:
I'm not sure that defaulting the accessibilityLabel to status makes sense since it's not localized. We should use a label that's passed in from the client instead.

Reply:
Good catch, I'll pass it from the Avatar

Reply:
Wouldn't badge status from the avatar still be a hardocded English string?

Reply:
No, I didn't put it to Avatar component, I passed it on AvatarTestPage. So the client will pass accessibilityLabel